src/Uno.UI.Runtime.Skia.Gtk/GLRenderSurface.cs

```diff
@@ -83,10 +83,12 @@ private void UnoGLDrawingArea_Render(object o, RenderArgs args)
 				_grContext = GRContext.CreateGl(glInterface);
 			}
 
+			var dpi = _dpi ?? 1f;
+			var scaledGuardBand = (int)(GuardBand * dpi);
```

Comment:
this could be updated inside `UpdateDpi` (which should not happen frequently)

Reply:
Will fix in subsequent update